src/game/piece.rs

```diff
@@ -168,10 +168,10 @@ impl Piece {
             ];
         } else if r#move == Movement::Down {
             return [
-                (self.positions[0].0 + 1, self.positions[0].1),
-                (self.positions[1].0 + 1, self.positions[1].1),
-                (self.positions[2].0 + 1, self.positions[2].1),
-                (self.positions[3].0 + 1, self.positions[3].1),
+                (self.positions[0].1, self.positions[0].1),
+                (self.positions[1].1, self.positions[1].1),
+                (self.positions[2].1, self.positions[2].1),
+                (self.positions[3].1, self.positions[3].1),
```

Comment:
hey uhhhhhh, I don't think this works quite like you were thinking it would

Reply:
I'm guessing you just used sed -i lol

Reply:
git ls-files, uhhh whoops

Reply:
you should checkout this branch and try to play the game with it. It's pretty cool xd